invenio_app_ils/ui/package.json

```diff
@@ -29,12 +29,15 @@
     "react-router-dom": "^5.0.1",
     "react-scripts": "3.1.2",
     "react-searchkit": "^0.13.0",
+    "react-show-more": "^2.0.0",
     "redux": "^4.0.4",
     "redux-devtools-extension": "^2.13.8",
     "redux-thunk": "^2.3.0",
     "semantic-ui-calendar-react": "^0.15.3",
+    "semantic-ui-css": "^2.4.1",
     "semantic-ui-less": "^2.4.1",
     "semantic-ui-react": "^0.88.0",
+    "serve": "^11.2.0",
```

Comment:
why do you need `serve`?

Reply:
we were checking with Zach options for split themes, I will remove it